kratos/tests/cpp_tests/processes/test_calculate_distance_to_skin_process.cpp

```diff
@@ -228,12 +228,13 @@ namespace Kratos {
 		// Compute distance
 		CalculateDistanceToSkinProcess<3>(volume_part, skin_part).Execute();
 
-		for (auto& node : volume_part.Nodes())
-			if (std::abs(node.GetSolutionStepValue(DISTANCE)) < 1.00e16) { // There are no propagation in this version so I avoid numeric_limit::max() one
-				auto distance = std::abs(node.Z() - 5.00);
-				KRATOS_CHECK_NEAR(node.GetSolutionStepValue(DISTANCE), distance, 1e-6);
+		for (auto& node : volume_part.Nodes()) {
+			if (node.Id() < 10) {
```

Comment:
why just 1e-4 as tolerance? cannot it be stricter?

Reply:
It has only 4 significant digits.

Reply:
the reference you mean? cannot we add more?

Reply:
I can add more zeros if you want.

Reply:
i thought it is an analytic value. if it is so, then let's be more precise, otherwise let's resolve this conversation

Reply:
I tried and it doesn't work. It says that "17.3205 is not equal to 17.3205"